.changeset/long-suns-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/vite-plugin-svelte': major
```

Comment:
Is major correct? Yes because repo is in pre release mode currently? Just want to flag to make sure it's not by accident

Reply:
Yeah I think marking as `patch` should be enough since it's unlikely to break users with this change. It'll help with the organization of the generated changelog.